jax/interpreters/xla.py

```diff
@@ -142,7 +142,14 @@ def xla_primitive_callable(prim, *abstract_args, **params):
 def primitive_computation(prim, *xla_shapes, **params):
   backend = params.get('backend', None)
   new_params = {k: params[k] for k in params if k != 'backend'}
-  c = xb.make_computation_builder("primitive_computation")  # TODO(name)
+  c = xb.make_computation_builder("primitive_computation_{}".format(prim.name))
+  c.SetOpMetadata(xc.OpMetadata(
+      op_type=prim.name,
+      op_name=str(core.new_jaxpr_eqn(
```

Comment:
nit: this op_name is pretty cryptic, consider moving into a function with a docstring (`pp_primitive()`?) or adding a comment that gives an example of what the output of this looks like. Or not, since this isn't hugely important.